disputer/disputer.js

```diff
@@ -61,13 +61,35 @@ class Disputer {
       });
 
       // Create the liquidation transaction
+      const dispute = this.empContract.methods.dispute(disputeableLiquidation.id, disputeableLiquidation.sponsor);
 
-      // TODO: compute the amount of collateral required to dispute and ensure the bot has enough to perform it.
+      // Simple version of inventory management: simulate the transaction and assume that if it fails, the caller didn't have enough collateral.
+      try {
+        await dispute.call({ from: this.account });
```

Comment:
why are you not specifying the `gasPrice` here?
```suggestion
        await dispute.call({ from: this.account, gasPrice: this.gasEstimator.getCurrentFastPrice()});
```